content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -309,6 +309,8 @@ Here's one scenario where you would set it. Suppose the Container listens on 127
 and the Pod's `hostNetwork` field is true. Then `host`, under `httpGet`, should be set
 to 127.0.0.1. If your pod relies on virtual hosts, which is probably the more common
 case, you should not use `host`, but rather set the `Host` header in `httpHeaders`.
+kubelet will make the probe connection at the node but not in pod. If you use the service
```

Comment:
kubelet can be running in a pod. 
It's better to remove `at the node but not in pod`

Reply:
Kubelet (sometimes) runs inside a Pod? That's puzzling.

Reply:
Ignore this. :smile:

Reply:
comment that the pod is the one want to do check.